docs/tutorials/excel-tutorial.md

```diff
@@ -296,9 +296,9 @@ In this step of the tutorial, you'll filter and sort the table that you created
 
 6. Within the `sortTable()` function, replace `TODO1` with the following code. Note:
 
-   - The code creates an array of `SortField` objects which has just one member since the add-in only sorts on the Merchant column.
+   - The code creates an array of `SortField` objects, which has just one member since the add-in only sorts on the Merchant column.
 
-   - The `key` property of a `SortField` object is the zero-based index of the column to sort-on.
+   - The `key` property of a `SortField` object is the zero-based index of the column to sort.
```

Comment:
I think we should be explicit about the sorting, since it's not just the column being sorted, but the entire table.

```suggestion
   - The `key` property of a `SortField` object is the zero-based index of the column used for sorting. The rows of the table are sorted based on the values in the referenced column.
```